### ISSUE TITLE
measure metric

### ISSUE THREAD

**Issue by bourne-3** [jedyang97/MTAG#1]
Hi jed,your job,MTAG,is very excelent. I had some question of it. 
I had download the code and run it.I found that the test set in the code may not be used?And perform the measure in the validation after the training step? It is also applied in the past method like Mult?

**Comment by jedyang97** [comment]
Hi @bourne-3, we are glad that you like our work!

The test set is not evaluated in the main training loop (only the validation set is evaluated during training). For evaluating on the test set, please use the `eval_model()` method provided in `main.py`. You will need to pass in a checkpoint that you want to use to evaluate. 

The numbers reported in the paper, including those from MulT, are on the test set.
